Cargo.toml

```diff
@@ -8,7 +8,7 @@ license = "MIT OR Apache-2.0"
 name = "secret-service"
 repository = "https://github.com/hwchen/secret-service-rs.git"
 edition = "2018"
-version = "2.0.1"
+version = "2.1.0"
```

Comment:
I believe this would need to be a `3.0.0` release in semver, no? This crate uses types from `zbus` in the public API (see the `Error` enum) which changed due to the `zbus` update. Additionally, the `Error::Crypto` variant had its data type changed.